src/EncompassRest/Contacts/BusinessContacts.cs

```diff
@@ -0,0 +1,161 @@
+﻿using EncompassRest.Utilities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace EncompassRest.Contacts
+{
+    public sealed class BusinessContacts
+    {
+        private const string s_apiPath = "encompass/v1/BusinessContacts";
+
+        #region Public Properties
+        public EncompassRestClient Client { get; }
+        #endregion
+
+        internal BusinessContacts(EncompassRestClient client)
+        {
+            Client = client;
+        }
+
+        public Task<BusinessContact> GetBusinessContactAsync(string contactId) => GetBusinessContactAsync(contactId, CancellationToken.None);
```

Comment:
Remove `Business` from the method names per the proposed API for example this would be changed to `GetContactAsync`.